src/containers/personalRequests/ducks/selectors.ts

```diff
@@ -0,0 +1,5 @@
+import { PersonalRequest } from './types';
+
+export const hasPendingRequest = (requests: PersonalRequest[]): boolean => {
+    return !!requests.find(request => request.status === 'PENDING');
```

Comment:
⛏️  I think .length > 0 would be more readable than !!

Reply:
We could also use an ormap here:
`return requests.some(request => request.status === 'PENDING')`